openshift-knative-operator/pkg/monitoring/monitoring.go

```diff
@@ -109,3 +121,74 @@ func InjectNamespaceWithSubject(resourceNamespace string, subjectNamespace strin
 		return nil
 	}
 }
+
+func LoadServingMonitoringPlatformManifests() ([]mf.Manifest, error) {
+	rbacPath := os.Getenv(servingSMRbacManifestPath)
+	if rbacPath == "" {
+		return nil, fmt.Errorf("failed to get the Serving sm rbac manifest path")
+	}
+	rbacManifest, err := mf.NewManifest(rbacPath)
+	if err != nil {
+		return nil, err
+	}
+	resourcePath := os.Getenv(servingSMResourceManifestPath)
+	if resourcePath == "" {
+		return nil, fmt.Errorf("failed to get the Serving sm resource manifest path")
+	}
+	smBlueprint, err := mf.NewManifest(resourcePath)
+	if err != nil {
+		return nil, err
+	}
```

Comment:
would it make sense to make these more generic ? 

E.g. other extensions could also add extra yaml, and would have to do the same?  

Not the end of the world, but just wondering

Reply:
I was thinking of doing this as a next step since I am planning to move eventing service monitors here as well.

Reply:
Perfect!